External/FEXCore/include/FEXCore/Config/Config.h

```diff
@@ -188,13 +188,27 @@ namespace Type {
       return &it->second.front();
     }
 
+    void Set(ConfigOption Option, const char *Data) {
```

Comment:
Wouldn't this overload's behavior also be covered by the string_view overload below?

Reply:
You would think that, but having a `Set` with only `std::string_view` and `fextl::string` causes string literals and `const char*` strings  to complain about ambiguous function calls.

Reply:
oh true, bleh